eventstream_rpc/include/aws/eventstreamrpc/EventStreamClient.h

```diff
@@ -71,32 +71,14 @@ namespace Aws
             EventStreamHeader(
                 const struct aws_event_stream_header_value_pair &header,
                 Crt::Allocator *allocator = Crt::g_allocator);
-            EventStreamHeader(const Crt::String &name, bool value);
```

Comment:
Don't they have a matching implementation that needs to be removed as well?

Reply:
No, I removed only methods with no implementation. If I accidentally removed a declaration for a method having definition, there'd be a compilation error.